core/sequencer/election.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sequencer
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/google/keytransparency/core/domain"
+	"github.com/google/keytransparency/core/mutator"
+	"github.com/google/trillian/util/election"
+)
+
+// Controller runs elections and controls receiver registration for a single domain.
+type Controller struct {
+	d                 *domain.Domain
+	factory           election.Factory
+	maxMastershipHold time.Duration
+}
+
+// NewController creates a new controller.
+func NewController(ctx context.Context, d *domain.Domain, factory election.Factory) *Controller {
+	return &Controller{
+		maxMastershipHold: 1 * time.Hour,
```

Comment:
Seems odd to hardcode this.

Reply:
I'll bubble it up one level. Do you think it's important enough for a flag?